hermes/crates/cardano-chain-follower/src/snapshot_id.rs

```diff
@@ -173,3 +173,97 @@ impl PartialOrd<u64> for SnapshotId {
         self.file.partial_cmp(other)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::point::*;
+    use super::*;
+
+    const TEST_DIR: &str = "test_snapshot_id";
```

Comment:
Don't create multiple test data directories.
We already have the `test_data` directory that holds test data, this data needs to be moved there.